src/legacy/core_plugins/kibana/public/discover/doc_table/_doc_table.scss

```diff
@@ -36,6 +37,7 @@ doc-table {
       font-size: 1.1em;
     }
   }
+  @include style_doc_source;
```

Comment:
Can you move the `include` to be the first line in the selector (above font-size)?

Reply:
of course, done